projects/storefrontstyles/scss/themes/calydon/layout/page-templates/_landing-page-2.scss

```diff
@@ -0,0 +1,13 @@
+%LandingPage2Template-calydon {
```

Comment:
same note as with the category page template styling.

Reply:
A ticket for this will be created.

Reply:
@tobi-or-not-tobi @kacperknapik Agreed and ticket #6711 created, it will be addressed during the `calydon-as-base` effort to avoid delaying the merge of this PR any further.